app/registries/foreman/plugin.rb

```diff
@@ -416,6 +421,10 @@ def action_scopes_hash_for(controller_class)
       @controller_action_scopes[controller_class.name] || {}
     end
 
+    def host_info_provider(provider_class)
```

Comment:
when is this supposed to be used? I though the registration is handled by `register_info_provider`